tests/basic_server.py

```
@@ -0,0 +1,110 @@
+                print(f'{nickname} changed nick to {new_nickname}')
+            elif decoded_message.lower() == 'bye':
+                close_connection(client)
+                break
+            else:
+                broadcast(f'{nickname}: {decoded_message}'.encode('ascii'))
+
+        except (ConnectionResetError, ConnectionAbortedError):
+            # Removing And Closing Clients on abrupt disconnect
+            close_connection(client)
+            break
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)